ush/wrappers/cheyenne/get_ics.qsub

```diff
@@ -0,0 +1,18 @@
+#! /bin/sh
+#PBS -A epic
+#PBS -l select=1:mpiprocs=1:ncpus=1:mem=2G
+#PBS -q regular
+#PBS -l walltime=00:30:00
+#PBS -j oe -o get_extrn_ics.log
+#PBS -N get_extrn_ics
+
+export GLOBAL_VAR_DEFNS_FP=$PWD'/../var_defns.sh'
+export USHdir=$PWD'/../../../ufs-srweather-app/ush'
```

Comment:
Suggesting to use
`export USHdir=$PWD'/../../ush'`

to provide nesting level similarly to the one used in line 18:
$USHdir"/../jobs/JREGIONAL_GET_EXTRN_MDL_FILES"